scipy/fft/_backend.py

```diff
@@ -0,0 +1,169 @@
+import scipy._lib.uarray as ua
+import scipy.fftpack as _fftpack
+from . import _pocketfft
+
+
+class _ScipyBackend:
+    """
+    The default backend for fft calculations
+
+    Notes
+    -----
+    We use the domain `numpy.scipy` because in the future `uarray` will treat
+    the domain as a heirarchy where `numpy.scipy` may call into a `numpy`
+    backend.
```

Comment:
Can you say what this implies? Like if there were a SciPy backend registered and a NumPy one registered, having the hierarchy as `numpy.scipy.fft` rather than just `scipy.fft` allows ...

FYI @peterbell10 for RST you probably want double-backticks here (render as code) as opposed to single backticks (default role, which for SciPy is autolink)